documentation/Installing-Clear-Containers-on-RHEL.md

```diff
@@ -0,0 +1,210 @@
+# Install the Clear Containers runtime on RHEL 7.3
+
+
+## Introduction
+
+Installing Clear Containers on RHEL is slightly more difficult than on other distributions mainly due to the fact that it does not provide current versions of gcc and glibc.
```

Comment:
Projects should be referred at least once in a document, in our case at least once per file, and an asterisk must be added. The asterisk makes reference to a legal disclaimer explaining that the name might be the trademark from someone else. A healthy rule is to use the project name just once with the asterisk and distinguishing it from the binary by using the extension or, as you suggested, backticks.